ambari-server/src/main/java/org/apache/ambari/server/agent/stomp/AgentConfigsHolder.java

```diff
@@ -16,12 +16,16 @@
  * limitations under the License.
  */
 package org.apache.ambari.server.agent.stomp;
+
 import java.util.List;
 import java.util.stream.Collectors;
 
+import javax.inject.Named;
```

Comment:
This is correct.
@zeroflag I made a mistake in `ConfigImpl` in my previous change to import `javax.inject.Named` instead of `com.google.inject.name.Named`. Could you please fix it here and there too? Thanks!

Reply:
fixed